enterprise/api/v1/urls.py

```diff
@@ -4,19 +4,25 @@
 
 from rest_framework.routers import DefaultRouter
 
+from django.conf import settings
 from django.urls import re_path
 
 from enterprise.api.v1 import views
 
 router = DefaultRouter()
-router.register("enterprise_catalogs", views.EnterpriseCustomerCatalogViewSet, 'enterprise-catalogs')
+
+# Endpoints are deprecated. See https://github.com/openedx/public-engineering/issues/61
+# This endpoint is no longer used, so it will be removed soon
+if getattr(settings, 'ENABLE_DEPRECATED_API_ENDPOINTS', True):
```

Comment:
I thought we should not use these endpoints by default, shouldn’t we?  I'm suggesting using the `False` as a default value if we should.

I'm would also prefer to use a more specific setting name, such as `ENABLE_DEPRECATED_ENTERPRISE_API_ENDPOINTS`

Reply:
I would stay with the current approach which inherit current behaviour and will change it only in case appropriate settings are set up